pkgs/applications/blockchains/framesh/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, fetchurl, appimageTools }:
+
+let
+  pname = "framesh";
+  version = "0.5.0-beta.19";
+  name = "${pname}-${version}";
```

Comment:
And this is unused now.

Reply:
it is used for symlinking the binary name, but can get rid of the variable declaration

Reply:
Maybe inline it then.